README.md

```diff
@@ -102,7 +113,12 @@ Using the object literal notation, Define an object called student and give it t
 
 Your Answer:
 ```js
-// write code here
+// var Student = new Student();
+    myStudent.age= Adult;
+    myStudent.gender= Female;
+    myStudent.name= Lyndsey;
+  console.log("Hi, my name is [myStudent.name].")
+
```

Comment:
Please try running this in the javascript console or repl.it.  Does it do what you expect?